backend/corpora/peaceportal/FIJI/fiji.py

```diff
@@ -0,0 +1,288 @@
+                full_path = op.join(directory, filename)
+                if extension != '.xml':
+                    logger.debug(self.non_xml_msg.format(full_path))
+                    continue
+                match = self.filename_pattern.match(name)
+                if not match:
+                    logger.warning(self.non_match_msg.format(full_path))
+                    continue
+                inscriptionID = match.groups()
+                yield full_path, {
+                    'inscriptionID': inscriptionID
+                }
+
+    def __init__(self):
+        super().__init__()
+        self.source_database.extractor = Constant(
+            value='Funerary Inscriptions of Jews from Italy (Utrecht University)'
+        )
+
+        self._id.extractor = XML(
+            tag=['teiHeader', 'fileDesc', 'titleStmt', 'title'],
+            toplevel=False,
+        )
+
+        self.url.extractor = Constant(
+            value=None
+        )
+
+        # the year is commented out: need to have not before / not after fields
+        # this is advisable since often we only roughly know the century
+        # self.year.extractor = XML(
+        #     tag=['teiHeader', 'fileDesc', 'sourceDesc',
+        #          'msDesc', 'history', 'origin', 'origDate'],
+        #     toplevel=False
+        # )
+
+        self.transcription.extractor = XML(
+            tag=['text', 'body', 'transcription'],
+            toplevel=False,
+            flatten=True
+        )
+
+        self.names.extractor = XML(
+            tag=['teiHeader', 'profileDesc',
+                 'particDesc', 'listPerson', 'person'],
+            flatten=True,
+            multiple=True,
+            toplevel=False,
+        )
+
+        self.sex.extractor = XML(
+            tag=['teiHeader', 'profileDesc',
+                 'particDesc', 'listPerson', 'person'],
+            attribute='sex',
+            multiple=True,
+            toplevel=False,
+        )
+
+        self.age.extractor = XML(
+            tag=['text', 'body', 'age'],
+            toplevel=False,
+            transform=lambda age: transform_age_integer(age)
+        )
+
+        self.country.extractor = Constant(
+            value='Italy'
+        )
+
+        self.settlement.extractor = XML(
+            tag=['teiHeader', 'fileDesc', 'sourceDesc',
+                 'msDesc', 'history', 'origin', 'provenance'],
+            toplevel=False,
+        )
+
+        self.material.extractor = XML(
+            tag=['text', 'body', 'material'],
+            toplevel=False,
+            transform=lambda x: categorize_material(x)
+        )
+
+        self.material_details = XML(
+            tag=['text', 'body', 'material'],
+            toplevel=False,
+        )
+
+        self.language.extractor = XML(
+            tag=['teiHeader', 'profileDesc', 'langUsage', 'language'],
+            toplevel=False,
+            multiple=True,
+            transform=lambda x: normalize_language(x)
+        )
```

Comment:
FIJI has values such as `Greek (transliterated)` - I don't think we'll be able to capture that in an iso code. For IIS, this is possible, so I'll add it there.

Reply:
Perhaps a more robust way to handle the FIJI categories would be to add a `script` field, too - this would allow us to express that an inscription is in Greek but written in Latin characters, or vice versa.

Reply:
Since I happened to be looking at the documentation for other reasons: IETF tags already supports including a script subtag, which is exactly for cases like that. I'd say it's more robust to include such information using a standardised system rather than come up with one of our own.